patch/kconfig-inclusions

```diff
@@ -58,5 +58,19 @@ CONFIG_MAX1363=m
 [arm64]
 
 [armhf]
+CONFIG_EEPROM_SFF_8436=m
+CONFIG_EEPROM_OPTOE=m
+CONFIG_I2C_MUX_GPIO=m
 
-
+[marvell-armhf]
+CONFIG_HZ_PERIODIC=y
+CONFIG_HZ_100=y
+CONFIG_HZ=100
```

Comment:
1.  Your [commit message](https://github.com/Azure/sonic-linux-kernel/pull/176/commits/3e00dbc2aaf8db8fe7b2450a0674f1529c06a01a) says, it’s in `linux-4.19.118/arch/arm/configs/mvebu_v7_defconfig`. But I do not find it there, and it’s also not in [Linux 5.10](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/arm/configs/mvebu_v7_defconfig?h=v5.10).
 
2.  You should only need `CONFIG_HZ_100=y`. `CONFIG_HZ=100` will be set by Linux’ Kconfig build system as a consequence.

Reply:
1. The changes were taken from Marvell private version of mvebu_v7_defconfig.
Hence removing it to be in sync with the upstreamed mvebu_v7_defconfig

2. Reverted the change to avoid conflicts with the upstreamed configuration